tdrs-backend/tdpservice/parsers/schema_defs/tanf/t6.py

```diff
@@ -0,0 +1,168 @@
+        Field(item="3A", name='CALENDAR_YEAR', type='number', startIndex=2, endIndex=6,
+              required=True, validators=[validators.month_year_yearIsLargerThan(1998)]),
+        Field(item="3B", name='CALENDAR_QUARTER', type='number', startIndex=6, endIndex=7,
+              required=True, validators=[validators.isInLimits(1, 4)]),
+        Field(item="4B", name='NUM_APPLICATIONS', type='number', startIndex=15, endIndex=23,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="5B", name='NUM_APPROVED', type='number', startIndex=39, endIndex=47,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="6B", name='NUM_DENIED', type='number', startIndex=63, endIndex=71,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="7B", name='ASSISTANCE', type='number', startIndex=91, endIndex=103,
+              required=True, validators=[validators.isInLimits(0, 999999999999)]),
+        Field(item="8B", name='NUM_FAMILIES', type='number', startIndex=123, endIndex=131,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="9B", name='NUM_2_PARENTS', type='number', startIndex=147, endIndex=155,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="10B", name='NUM_1_PARENTS', type='number', startIndex=171, endIndex=179,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="11B", name='NUM_NO_PARENTS', type='number', startIndex=195, endIndex=203,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="12B", name='NUM_RECIPIENTS', type='number', startIndex=219, endIndex=227,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="13B", name='NUM_ADULT_RECIPIENTS', type='number', startIndex=243, endIndex=251,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="14B", name='NUM_CHILD_RECIPIENTS', type='number', startIndex=267, endIndex=275,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="15B", name='NUM_NONCUSTODIALS', type='number', startIndex=291, endIndex=299,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="16B", name='NUM_BIRTHS', type='number', startIndex=315, endIndex=323,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="17B", name='NUM_OUTWEDLOCK_BIRTHS', type='number', startIndex=339, endIndex=347,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="18B", name='NUM_CLOSED_CASES', type='number', startIndex=363, endIndex=371,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+    ],
+)
+
+s3 = RowSchema(
```

Comment:
@elipe17 one immediate issue i see here (which i think is due at least in part to [this incomplete example](https://user-images.githubusercontent.com/63075587/257940006-b08aeda4-3b62-4580-86cf-fe0f1dd60886.PNG) i added in the ticket), is that its not possible to tell which month is associated with each record. 

_DAC_
![2649p3](https://github.com/raft-tech/TANF-app/assets/63075587/ca4ab10f-3d2e-4bdb-b986-5675a53dca27)

_notebook_
![2649p4](https://github.com/raft-tech/TANF-app/assets/63075587/2335adf9-5507-4bde-8db4-6d2c74a5d130)